src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -15,7 +15,7 @@ namespace Dynamo.DocumentationBrowser
     /// </summary>
     public class DocumentationBrowserViewExtension : IViewExtension, ILogSource
     {
-        private ViewLoadedParams viewLoadedParams;
+        private ViewLoadedParams viewLoadedParamsRefenrence;
```

Comment:
Typo: There is an extra `n` in `Refenrence`

Reply:
Thanks, fixing